sites/docs/usage/env.rst

```diff
@@ -139,7 +139,7 @@ as password prompts, "What host to connect to?" prompts, fabfile invocation of
 session will always terminate cleanly instead of blocking on user input forever
 when unforeseen circumstances arise.
 
-.. versionadded:: 1.1
+.. versionadded:: 1.10.3
```

Comment:
EIther Github's diffing is off or you shouldn't even be touching this line at all? Your changes don't affect `abort_on_prompts`, right?